src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/BookingIntoPrisonService.kt

```diff
@@ -97,49 +97,51 @@ class BookingIntoPrisonService(
     offender.bookings.forEach(OffenderBooking::incBookingSequence)
 
     return offenderBookingRepository.save(
-      OffenderBooking()
-        .withBookingBeginDate(receiveTime)
-        .withBookNumber(bookNumber)
-        .withOffender(offender)
-        .withLocation(prison)
-        .withAssignedLivingUnit(cellOrReception)
```

Comment:
There's nothing wrong with `@With`, just standardising the approach to use builders.